Makefile

```diff
@@ -108,6 +111,14 @@ test_pkg_nodejs: get_schemas
 test_pkg_python: get_schemas
 	@cd pkg && $(GO_TEST) ${PKG_CODEGEN_PYTHON}
 
+test_pkg_dotnet: get_schemas
+# invoked as part of "test_pkg_rest", listed separately to update codegen just for dotnet
```

Comment:
Same comment copied to nodejs and python above?

Reply:
`test_pkg_rest` does not invoke python or nodejs, it runs everything but those.

```
PROJECT_PKGS    := $(shell cd ./pkg && go list ./... | grep -v -E '^(${PKG_CODEGEN_NODEJS}|${PKG_CODEGEN_PYTHON})$$')
```

Reply:
Maybe comment that instead 😨 that's very non-obvious

Reply:
Adding that and then merging!